src/countervalues/logic.js

```diff
@@ -380,33 +398,33 @@ function generateCache(
     // if autofillGaps is on, shifting daily gaps (hourly don't need to be shifted as it automatically fallback on a day rate)
     const now = Date.now();
     const oldestTime = oldestDate.getTime();
-    let shiftingValue = map[oldest];
+    let shiftingValue = map.get(oldest) || 0;
```

Comment:
`map.get(oldest) ?? 0` to be precise

Reply:
i'm not super confident by this idea. sometimes API returned null in past and we may want to be just "falsy case" fallback.